components/App/App.jsx

```diff
@@ -2,7 +2,7 @@ const React = require('react')
 const Filter = require('../Filter')
 const TaskList = require('../TaskList')
 
-class Main extends React.Component {
+class App extends React.Component {
```

Comment:
I had originally gone away from app.jsx because I find it a little confusing. There's main.js, snoozer.js, and app.jsx. All are kind of entry points for the app. This component in particular is the main page of the app, but only for signed-in users.

I'm okay sticking with App for now, but I think this could be a little clearer. That could wait till we have the concept of logging in with your GitHub account, though, and have to distinguish between the signed-in versus signed-out landing page.